### ISSUE TITLE
[TEST] Multi-task Learning에서 loss 계산 방식 테스트

### ISSUE THREAD

**Issue by 2018007956** [boostcampaitech6/level1-imageclassification-cv-14#47]
## Background
- Multi-task Learning 시 손실값을 합산하여 그대로 사용하는 경우와 정규화된 손실값을 사용하는 경우, 특정 클래스에 가중치를 준 경우 비교

## To do
- [x] loss/task수 등으로 정규화하여 loss update 해보기
`loss = (loss_mask + loss_gender + loss_age)/3`
- [x] 어떤 태스크에서 약한지 파악 후 해당 태스크의 loss에 가중치 부여하기
- data augmentation에서 파라미터를 고정시켜 모든 세팅을 동일하게 한 후 테스트 - After #50
